dlrover/python/elastic_agent/torch/training.py

```diff
@@ -730,6 +734,37 @@ def stop_executor(self):
         """Shutdown the executor to save the checkpoint."""
         self._save_ckpt_executor.shutdown(wait=False)
 
+    def sync_training_ports(self):
```

Comment:
It is better to implement a separate class for HPU processes.

Reply:
We need to release this function asap. We will refactor this in future pr.